packages/console/src/components/RoleEntitiesTransfer/components/TargetEntityItem/index.module.scss

```diff
@@ -3,22 +3,21 @@
 .item {
   display: flex;
   align-items: center;
-  padding: _.unit(2.5) _.unit(4);
+  padding: _.unit(2) _.unit(3) _.unit(2) _.unit(4);
```

Comment:
question: why do we need change 3 components for this. the original code can be better?

Reply:
Yeah, had the same question, and have created a task to refactor the code later.
https://linear.app/silverhand/issue/LOG-7102/extract-re-usable-transfer-component-for-rolescopestransfer